lib/graphql/language/generation.rb

```diff
@@ -116,7 +130,7 @@ def generate(node, indent: "")
         when Array
           "[#{node.map { |v| generate(v) }.join(", ")}]"
         when Hash
-          "{ #{node.map { |k, v| "#{k}: #{generate(v)}" }.join(", ")} }"
+          "{#{node.map { |k, v| "#{k}: #{generate(v)}" }.join(", ")}}"
```

Comment:
Am I right that this whitespace change was added so that our dump matches the "kitchen sink" example from graphql.js ? It's good by me, I just want to make sure I understand it before adding noise to peoples' diffs :P